common/components/forms/PositionList.jsx

```diff
@@ -129,40 +211,48 @@ class PositionList extends React.PureComponent<Props, State> {
           message="Do you want to delete this position?"
           onSelection={this.confirmDelete.bind(this)}
         />
+
+        <ConfirmationModal
+          showModal={this.state.showHideModal}
+          message="Do you want to hide this position?"
```

Comment:
Can we get some explanatory text from design on this? Let the user know what hiding means -- it's not deleted but also not shown to the public until you choose to unhide it, whatever. It's not in the specification right now so it's not blocking but I think we should consider it?

Reply:
Agree in principle, but think it's something design should chime in on.

Reply:
Sounds good.